src/alerts/flash-refactor.scss

```diff
@@ -0,0 +1,133 @@
+// stylelint-disable selector-max-type
+
+// Flash alert
+
+.Flash {
+  position: relative;
+  display: grid;
+  padding: $spacer-2;
+  color: var(--color-fg-default);
+  background-image: linear-gradient(var(--color-accent-subtle), var(--color-accent-subtle));
+  border: $border-width $border-style var(--color-accent-muted);
+  border-radius: $border-radius;
+  grid-auto-flow: column;
+  grid-template-areas: 'visual message actions close';
+  grid-template-columns: min-content 1fr min-content min-content;
+  grid-template-rows: min-content;
+
+  // `sm` breakpoint variantion
+  @media (max-width: #{map-get($breakpoints, 'sm') - 0.02px}) {
+    grid-template-areas:
+      'visual message close'
+      '  .    actions actions';
+    grid-template-columns: min-content 1fr min-content;
+    grid-template-rows: min-content min-content;
+
+    .Flash-actions {
+      margin: $spacer-2 0 0 $spacer-2;
+    }
+  }
+
+  // Elements
+
+  .Flash-visual {
+    display: grid;
+    // stylelint-disable-next-line primer/spacing
+    padding: calc((#{$spacer-5} - 20px) / 2) $spacer-2;
```

Comment:
Is there a specific reason to use `calc()` here and 2 more times below? I think the browser will see it as:

```css
padding: calc((32px - 20px) / 2) 8px;
```

but because they are static `px` values it won't change. So we could also just use the calculated `px` value and probably improve performance a tiny bit: 

```scss
padding: 6px $spacer-2;
```

If it was to get around the `stylelint-disable`, @jonrohan patched that recently. 😉

Reply:
@simurai I'm getting prepared for the use of [functional spacing design tokens](https://docs.google.com/document/d/1Yc5WKc0R0mAQFDPOsS9HgLM7ZFB0lNJHWeeycbi5mvY/edit#heading=h.yuz1kk21vxj2) (link only available for hubbers). `$spacer-5` should be `--gh-control-medium-size`, and `20px` `--gh-text-body-medium-lineBoxHeight`. This calculation adds any extra padding needed so the 16px octicon gets vertically centered with the first line of text, even in case there are multiple lines.

I'll add this as a comment in the file. 🙇

Reply:
> I'm getting prepared for the use of [functional spacing design tokens](https://docs.google.com/document/d/1Yc5WKc0R0mAQFDPOsS9HgLM7ZFB0lNJHWeeycbi5mvY/edit#heading=h.yuz1kk21vxj2)

ahhh.. ok ok, in that case let's keep as is for easier updating later. 👍